composer.json

```diff
@@ -7,6 +7,7 @@
   "license": "GPL-2.0+",
   "minimum-stability": "dev",
   "require": {
+    "ext-gd": "*",
```

Comment:
Seems a little wild. Perhaps some restraint?

Reply:
I realize why this is a `*`. it is basically requiring a php extension. and since php versions are different version changes.